src/MySQLApplication.php

```diff
@@ -5,9 +5,28 @@
 namespace Keboola\DbWriter;
 
 use Keboola\Csv\CsvFile;
+use Keboola\DbWriter\Configuration\MySQLActionConfigRowDefinition;
+use Keboola\DbWriter\Configuration\MySQLConfigDefinition;
+use Keboola\DbWriter\Configuration\MySQLConfigRowDefinition;
 
 class MySQLApplication extends Application
 {
+
+    public function __construct(array $config, Logger $logger)
+    {
+        $action = !is_null($config['action']) ?: 'run';
+        if (isset($config['parameters']['tables'])) {
+            $configDefinition = new MySQLConfigDefinition();
+        } else {
+            if ($action === 'run') {
+                $configDefinition = new MySQLConfigRowDefinition;
+            } else {
+                $configDefinition = new MySQLActionConfigRowDefinition();
+            }
```

Comment:
myslim ze to je dobre, ale uz nechapu proc to jsme nedelali taky v snowflake writer

Reply:
protože v testech se `testConnection` provádí celou konfigurací (včetně `tables`) - už to upravuji

Reply:
kul, tak treba davame tam na ten row action config `->ignoreExtraKeys(false)` protoze nejsem jistej stopro co posla uicko

Reply:
ok je tam :+1: